src/style.css

```diff
@@ -0,0 +1,94 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+
+}
+
+.block {
+  text-align: center;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
indicate font-family in html or body tag

Reply:
ok, got it :)